### PR TITLE
[Invitation by email] Handle correctly Id server consent

### PR DESCRIPTION
**Pull request by ClaireGizard**
#392

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/userdirectory/UserListViewModel.kt [hunk 1]
```diff
@@ -85,6 +85,11 @@ class UserListViewModel @AssistedInject constructor(
     }
 
     init {
+        // Tchap: Force user consent to true, we don't want to display the consent banner
+        if (!session.identityService().getUserConsent()) {
+            session.identityService().setUserConsent(true)
+        }
+
```
Comment (giomfo) [inline vector/src/main/java/im/vector/app/features/userdirectory/UserListViewModel.kt:92]: @Florian14 can you please confirm we need to force the consent here too?

**Review by giomfo — APPROVED** [review]
SGTM
I let @Florian14 approve it too

File: vector/src/main/java/im/vector/app/features/userdirectory/UserListViewModel.kt (hunk 1 above)
Comment (Florian14) [reply vector/src/main/java/im/vector/app/features/userdirectory/UserListViewModel.kt:92]: This one is necessary but I am not sure the other one is necessary as we don't display the contact book in Tchap

Comment (ClaireGizard) [reply vector/src/main/java/im/vector/app/features/userdirectory/UserListViewModel.kt:92]: The ContactBook is not visible in tchap. We don't need to set the userConsent to true.

**Review by Florian14 — APPROVED** [review]
LGTM

**Pull request merged**